assets/js/form.js

```diff
@@ -0,0 +1,41 @@
+// whole form
+// view
+var FormView = Backbone.View.extend({
+  template: AppTemplates.form,
+
+  el: '#target',
+
+  initialize: function() {
+    this.render();
+  },
+
+  events: {
+    'form .submit': 'submitForm'
```

Comment:
You have this backwards.

The format for event listeners is `'eventname selector': 'functionName'`. `form` would need to be your selector and you should be listening for a `submit` (not `.submit`) event.

Right now you are listening for a `form` event (which doesn't exist) on a piece of HTML with a class of `submit`.